source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet

```diff
@@ -53,12 +53,12 @@ local proxyDescription = 'If they had no usual address one year ago, state the a
 local additionalAnswerOption = [
   {
     label: {
-      text: 'Same address as {first_person}',
+      text: 'Same as {first_person} address one year ago',
```

Comment:
Missing the `'s`

Reply:
Changed the placeholder name so it is more clear now